packages/grafana-ui/src/components/Tags/Tag.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC } from 'react';
+import { cx, css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { useTheme } from '../../themes';
+import { getTagColorsFromName } from '../../utils';
+
+interface TagProps {
+  /** Name of the tag to display */
+  name: string;
+  onClick?: (name: string) => any;
+  /** Custom styles for the wrapper component */
+  className?: string;
+  [key: string]: any;
```

Comment:
why the [key: string]: any here? Makes it sort of untyped :) 

Wonder if it's better to skip the onClick & className and just have a forwardRef<HTMLElement, Props>

Reply:
Ah, good catch! Was using it so the compiler doesn't complain about `className`, but forgot to remove it after.

Reply:
One option would probably be to just omit the 'onClick':
```
interface Props extends Omit<HTMLAttributes<HTMLElement>, 'onClick'> {
  /** Name of the tag to display */
  name: string;
  onClick?: (name: string) => any;
}
```